Add pages for submission service (#6)
expo/fyi: missing-privacy-policy.md

# Missing Privacy Policy

#### 🤔 What Happened

> If you're submitting your Android app to Google Play Store using the `expo upload:android` command or Google Play console, you may encouter an error saying you need to have a privacy policy set for your app.

Reviewer: ```suggestion
If you're submitting your Android app to Google Play Store using the `expo upload:android` command or Google Play console, you may encounter an error saying you need to have a privacy policy set for your app. It would look something like this on the Google Play Console:
```

[<img src="./assets/missing-privacy-policy/01-privacy-policy-error.png" width="800" />](./assets/missing-privacy-policy/01-privacy-policy-error.png)

> If you were trying to submit your app with `expo upload:android` you may have seen the following error:

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> > The app has permissions that require a privacy policy set for the app.

Reviewer: ```suggestion
```
